Cargo.toml

```diff
@@ -40,6 +40,8 @@ forge = { git = "https://github.com/foundry-rs/foundry", branch = "master" }
 # see https://github.com/dojoengine/dojo/issues/563
 dojo-test-utils = { git = "https://github.com/jobez/dojo", branch = "custom-steps-and-get-block-potential-fix"  }
 [patch.crates-io]
+# patched for quantity U256 responses <https://github.com/recmo/uint/issues/224>
+ruint = { git = "https://github.com/paradigmxyz/uint" }  
```

Comment:
When we include any rust git dependency, we have to make sure we observe the project's patches, as it can create subtle yet drastic differences in the behavior in the libraries usage!

https://github.com/paradigmxyz/reth/blob/main/Cargo.toml#L74

see:
https://github.com/foundry-rs/foundry/issues/5303#issuecomment-1628822918

Reply:
nice catch, v cursed, i dont understand why projects dont inherit patches?

Reply:
would be cool to find out as a team, philosophically